pkg/pool/missing-date.go

```diff
@@ -0,0 +1 @@
+package pool
```

Comment:
<!--copyright-bot-->

Missing copyright header, expected comment of the form:

    /*
    Copyright 2019 Something something authors.
    ...
    */